parsons/controlshift/controlshift.py

```diff
@@ -0,0 +1,76 @@
+from parsons.etl.table import Table
+from parsons.utilities import check_env
+from parsons.utilities.oauth_api_connector import OAuth2APIConnector
+
+
+class Controlshift(object):
+    """
+    Instantiate the Controlshift class. Requires an API Application integration.
```

Comment:
Line too long (79-character pep8 limit)

Reply:
According to our contrib guidelines and .flake8 config, we set a max-line-length to 100. https://github.com/move-coop/parsons/blob/master/CONTRIBUTING.md#linting

Our CI is also linting according to that rule...